src/main/java/org/jenkinsci/plugins/fodupload/controllers/DastScanController.java

```diff
@@ -90,6 +88,22 @@ public PatchDastFileUploadResponse PatchDynamicScan(PatchDastScanFileUploadReq r
                     temp = File.createTempFile("worflowdriven", requestModel.releaseId);
                     fileName = "WorkflowDriven.webmacro";
                     break;
+                case OpenAPIDefinition:
+                    temp = File.createTempFile("openAPIDefinition", requestModel.releaseId);
+                    fileName = "OpenAPIDefinition.json";
+                    break;
+                case GRPCDefinition:
+                    temp = File.createTempFile("grpcDefinition", requestModel.releaseId);
+                    fileName = "GRPCDefinition.proto";
+                    break;
+                case GraphQLDefinition:
+                    temp = File.createTempFile("graphQLDefinition", requestModel.releaseId);
+                    fileName = "GraphQLDefinition.json";
+                    break;
+                case PostmanCollection:
+                    temp = File.createTempFile("postmanCollection", requestModel.releaseId);
+                    fileName = "PostmanCollection.json";
+                    break;
```

Comment:
please Add try catch block to delete the temp file.   in catch ignore the error.

Reply:
As discussed it exist